zone/entity/match_players_list.php

```diff
@@ -182,4 +182,28 @@ private static function get_min_rating_of_all(match_players_list ...$players_lis
             return $list->get_min_rating();
         }, $players_lists));
     }
+
+    private static function get_mean_rating(match_players_list ...$players_lists): int
+    {
+       $value = 0;
+       $number = 0;
+       foreach ($players_lists as $list) {
+        $value += $list->get_total_rating();
+        $number += $list->length();
+       }
+       return $value/$number;
```

Comment:
preferred: `return $value / $number;`

Reply:
ok, sry. In java `int` / `int` is automatically rounded and casted to an `int`. The intention was that the result is in fact rounded to an `int`. What does php do here?

Reply:
@Des- yes, that is what would happen in php too. i was not sure wether that was intentional here ^^